packages/ui/components/alert/Alert.tsx

```diff
@@ -2,13 +2,18 @@ import { CheckCircleIcon, ExclamationIcon, InformationCircleIcon, XCircleIcon }
 import classNames from "classnames";
 import { ReactNode } from "react";
 
+import { Icon } from "../icon";
+
 export interface AlertProps {
   title?: ReactNode;
+  // @TODO: Message should be children, more flexible?
```

Comment:
I think this component in general can still use some improvements (it's also not really according to design). So I left some todo's for when we are updating this component.